compose/tests/test_docker_compose.py

```diff
@@ -9,8 +10,14 @@
 from testcontainers.core.waiting_utils import wait_for_logs
 
 ROOT = os.path.dirname(__file__)
+"""
 
 
+def test_can_spawn_service_via_compose():
+    assert 2 == 2
+
+
+"""
```

Comment:
Can we use a pytest feature to skip these tests?

```suggestion
"""
@pytest.mark.skip(reason="docker compose support has been removed to fix dependency issues in the build")
"""
```

Reply:
the danger with that is that empty test should be reported as a failure (imo its a bug if pytest passes with 0 tests) - next/soon pr should be removal of docker-compose altogether, anyway right?

Reply:
looking into this locally

Reply:
ok, pushed fix for this one